<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetDownloads</class>
 <widget class="QMainWindow" name="WidgetDownloads">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Downloads</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/Resource/Generic/Download.png</normaloff>:/Resource/Generic/Download.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableView" name="tableViewDownloads">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>14</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBarControls">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPreview"/>
   <addaction name="actionPause"/>
   <addaction name="actionCancel"/>
   <addaction name="actionOpenTorrent"/>
   <addaction name="separator"/>
   <addaction name="actionMarkAsJunk"/>
   <addaction name="actionShowMonitor"/>
  </widget>
  <widget class="QToolBar" name="toolBarFilter">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClearCompleted"/>
  </widget>
  <action name="actionFilterAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Download.png</normaloff>:/Resource/Generic/Download.png</iconset>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <property name="toolTip">
    <string>All</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Media/PlayMedia.png</normaloff>:/Resource/Media/PlayMedia.png</iconset>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="toolTip">
    <string>Preview</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Media/PauseMedia.png</normaloff>:/Resource/Media/PauseMedia.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Cancel.png</normaloff>:/Resource/Generic/Cancel.png</iconset>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="toolTip">
    <string>Cancel</string>
   </property>
  </action>
  <action name="actionMarkAsJunk">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Library/Junk.png</normaloff>:/Resource/Library/Junk.png</iconset>
   </property>
   <property name="text">
    <string>Mark As Junk</string>
   </property>
   <property name="toolTip">
    <string>Mark As Junk</string>
   </property>
  </action>
  <action name="actionShowMonitor">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Graph.png</normaloff>:/Resource/Generic/Graph.png</iconset>
   </property>
   <property name="text">
    <string>Show Monitor</string>
   </property>
   <property name="toolTip">
    <string>Show Monitor</string>
   </property>
  </action>
  <action name="actionOpenTorrent">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Networks/BitTorrent.png</normaloff>:/Resource/Networks/BitTorrent.png</iconset>
   </property>
   <property name="text">
    <string>Open Torrent</string>
   </property>
   <property name="toolTip">
    <string>Open Torrent</string>
   </property>
  </action>
  <action name="actionClearCompleted">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Clear.png</normaloff>:/Resource/Generic/Clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Completed</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/DownloadPreview.png</normaloff>:/Resource/Generic/DownloadPreview.png</iconset>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
  </action>
  <action name="actionAddToPlaylist">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Media/AddPlaylist.png</normaloff>:/Resource/Media/AddPlaylist.png</iconset>
   </property>
   <property name="text">
    <string>Add To Playlist</string>
   </property>
  </action>
  <action name="actionViewReviews">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Check.png</normaloff>:/Resource/Generic/Check.png</iconset>
   </property>
   <property name="text">
    <string>View Reviews</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Help.png</normaloff>:/Resource/Generic/Help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionFindMoreSources">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Search.png</normaloff>:/Resource/Generic/Search.png</iconset>
   </property>
   <property name="text">
    <string>Find More Sources</string>
   </property>
  </action>
  <action name="actionVirusTotalCheck">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Network/virustotal-virus-scanner.png</normaloff>:/Resource/Network/virustotal-virus-scanner.png</iconset>
   </property>
   <property name="text">
    <string>Virus Total Check</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
